lib/assessment.js

```diff
@@ -8,15 +8,20 @@
 // Question 1
 // Write a callback for reduce that selects
 // the minimum value in the array
-var min = function min() {
-
+var min = function min(memo, current) {
+  if (current < memo) {
+    memo = current;
+  }
+  return memo;
 };
 
 // Question 2
 // Write a callback for reduce that produces
 // the sum of the values in the array
-var sum = function sum() {
-
+var sum = function sum(memo, current) {
+  memo = memo || 0;
+  memo += current;
+  return memo;
```

Comment:
Correct callback, but would be better if it didn't assign to one of its parameters. 1